sites.json

```diff
@@ -2350,6 +2370,16 @@
     },
 
     {
+        "name": "gPodder",
```

Comment:
This should be listed between GoSquared and Grailed, not between GoPetition and GoSquared.